From ab9d0d2f57ed84c01513053b88587dac82d9ed66 Mon Sep 17 00:00:00 2001
From: Sitaram Chamarty <sitaram@atc.tcs.com>
Date: Thu, 17 Nov 2011 18:49:24 +0530
Subject: [PATCH] a couple of very minor oopsies

(1) c5f342a forgot authkeys can have blank lines also

(2) f0cedeb ended gl-system-install on a non-0 ret code

both caught by matthewtdawson at gmail
---
 src/gl-system-install |    4 +++-
 src/sshkeys-lint      |    2 +-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/gl-system-install b/src/gl-system-install
index 0eec005..d7df7e4 100755
--- a/src/gl-system-install
+++ b/src/gl-system-install
@@ -108,8 +108,10 @@ gl-setup is not in your \$PATH.
 $path_advice"
 
 which=`dirname $which`
-[ "$which" != "$gl_bin_dir" ] && die "                ***** WARNING *****
+[ "$which" = "$gl_bin_dir" ] || die "                ***** WARNING *****
 $which precedes $gl_bin_dir in your \$PATH,
 and it *also* contains gl-setup.  This is almost certainly going to confuse
 you or me later.
 $path_advice"
+
+exit 0
diff --git a/src/sshkeys-lint b/src/sshkeys-lint
index bcb8f7b..96c027f 100755
--- a/src/sshkeys-lint
+++ b/src/sshkeys-lint
@@ -119,7 +119,7 @@ sub ak_comment {
     $in_gl_section = 1 if /^# gitolite start/;
     $in_gl_section = 0 if /^# gitolite end/;
     die "gitosis?  what's that?\n" if /^#.*gitosis/;
-    return /^\s*#/;
+    return /^\s*(#|$)/;
 }
 
 sub fprint {
-- 
1.7.5.4

